### DISCUSSION TITLE
All albums from a specific artist not shown

### DISCUSSION THREAD

**Discussion by avalx** [jcorporation/myMPD#726] [Q&A] [answered]
myMPD version: [e.g. 8.1.0]

Maybe not a real bug but an enhacement suggestion.

When I try to list all albums from a particular artist not all albums are shown. 

**To Reproduce**

1. Click on an artist name in Albums library view
2. Click on 'Show all albums from artist'
3. Only some albums are shown

**Expected behavior**

As much as I can see this function only checks for the tag 'Album Artist' to list all albums from this artist. The same thing happens when I click on the artist name in the expanded album view (the view with album art and songs listed). I hardly ever configure this tag when modifying the metadata in my music collection. I only check that the 'Artist' tag is configured properly. I would suggest that also the tag 'Artist' is considered when searching/displaying all albums from some artist(s).

**Screenshots**

![Clipboard01](https://user-images.githubusercontent.com/22810741/163117043-d9556f0a-63b5-4200-90c1-5dda3dc9412b.jpg)

![Clipboard02](https://user-images.githubusercontent.com/22810741/163117064-236cfd5b-85db-4e7e-b826-02cc95e2e556.jpg)

**Server plattform (please complete the following information):**
 - OS: [e.g. DietPI] Fedora
 - OS version [e.g. 6.26] 35
 - Plattform: [e.g. x86_64] 
 - MPD version: [e.g. 0.21.16] 	0.23.5

**Client plattform (please complete the following information):**
 - Device: [e.g. iPhone6, Laptop] PC
 - OS: [e.g. iOS, Windows 10, Ubuntu Linux 19.10] Windows
 - Browser [e.g. chrome, safari] all
 - Browser version [e.g. 22] latest

**Additional context**

**Comment by jcorporation** [comment]
myMPD uses the albumartist tag if it is available and falls back to artist tag if not available. You should disable the albumartist tag if you do not maintain it. Mixing this two tags is not a good idea.

**Comment by avalx** [comment]
> myMPD uses the albumartist tag if it is available and falls back to artist tag if not available. You should disable the albumartist tag if you do not maintain it. Mixing this two tags is not a good idea.

I'm not sure how to do this - disable the albumartist tag? I've also tried this with some other artists and never got a listing of any albums of the specific artist - it is always an empty list. For example I have a few albums by this artist and the list is still empty!

![Clipboard01](https://user-images.githubusercontent.com/22810741/163257059-f429428f-a6c9-472c-87d2-3ecd9d6f7dc8.jpg)

**Comment by jcorporation** [comment, marked as answer]
Edit mpd.conf and change the meta_to_use option or disable the tag in the myMPD settings.

Best way is to add the albumartist tag to all your files. Especially compilations can not be grouped without the albumartist tag.

**Reply by avalx** [reply]
Thanks for the tip about mpd.conf. I'll see what I can do with that.

Actually the only place that I already use 'Album Artist' tag are the compilation albums. Adding this to my whole library will be a major task so I'll have to check if there is a way to automate this. I didn't do this up until now because other media library apps did not request this tag to be present, but from now on I'll take care also about this as well.
